packaging/irods.service.systemd

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
our other template files are shipped/packaged with `.template`

Reply:
Renamed to use `.template`.